app/_hub/kong-inc/service-protection/how-to/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+nav_title: # Set a title for the navigation menu, for example "Using the example-name plugin"
+title: # Set a title to appear at the top of the page
+---
+<!-- Describe how to use your plugin -->
```

Comment:
Enable service protection plugin with Kong Admin API 

Make the following request:

> curl -X POST http://localhost:8001/services/{serviceName|Id}/plugins/ \
>     --header "accept: application/json" \
>     --header "Content-Type: application/json" \
>     --data '
>     {
>   "name": "service-protection",
>   "config": {
>     "limit": [
>       5
>     ],
>     "window_size": [
>       60
>     ],
>     "sync_rate": -1,
>     "namespace": "example_namespace",
>     "strategy": "redis",
>     "hide_client_headers": false
>   }
> }
>     '


Replace serviceName|Id with the id or name of the service that this plugin configuration will target.

Reply:
Ah, I see that in your engineering brief, you wrote up examples using this plugin together with RLA. I'm going to use those :)

Reply:
That sounds good, thank you.